relay-redis/src/real.rs

```diff
@@ -53,7 +62,10 @@ impl ConnectionLike for ConnectionInner<'_> {
                 secondaries,
             } => thread::scope(|s| {
                 for connection in secondaries {
-                    s.spawn(move || log_secondary(connection.req_packed_command(cmd)));
+                    let result = thread::Builder::new().spawn_scoped(s, move || {
+                        log_secondary_redis_error(connection.req_packed_command(cmd));
+                    });
+                    log_secondary_thread_error(result);
```

Comment:
Possibly can do a `spawn_secondary_thread` instead:

```suggestion
                    spawn_secondary_thread(s, move || {
                        log_secondary_redis_error(connection.req_packed_command(cmd));
                    });
```

And in the `spawn_secondary_thread` spawn via the builder and log the error.